client/src/app/licitacoes/licitacoes-detalhar/licitacoes-detalhar.component.ts

```diff
@@ -34,8 +36,22 @@ export class LicitacoesDetalharComponent implements OnInit {
       .pipe(takeUntil(this.unsubscribe))
       .subscribe(licitacao => {
         this.licitacao = licitacao;
-        this.itens = licitacao.itensLicitacao;
+        this.licitacao.itensLicitacao = licitacao.itensLicitacao;
       });
   }
 
+  get itensPaginate(): ItensLicitacao[] {
+    if (this.licitacao.itensLicitacao) {
```

Comment:
```suggestion
    if (this.licitacao && this.licitacao.itensLicitacao) {
```
Acho que se não checar o this.licitacao tbm dá erro.